File: FRONTEND/SisconFrontEnd/SisconFrontEnd/src/app/models/Predio.ts
export class Predio {

  id: number;
  nomePredio: string;
  rua: string;
  numero: number;
  bairro: string;
  cidade: string;
  estado: string;
  pais: string;
  cep: string
  dataConstrucao: string
  numAndares: string

}

export class PredioPayload {
  data: Predio[];
}

export class PredioPayloadUniqueResult {
  data: Predio;
}

File: BACKEND/Siscon_API_Backend/Siscon_API_Backend/Models/Predio.cs
namespace Siscon_API_Backend.Models
{
    public class Predio
    {
        public long Id { get; set; }
        public string NomePredio { get; set; }
        public string Rua { get; set; }
        public int? Numero { get; set; }
        public string Bairro { get; set; }
        public string Cidade { get; set; }
        public string Estado { get; set; }
        public string Pais { get; set; }
        public string CEP { get; set; }
        public DateTime? DataConstrucao { get; set; }
        public int? NumAndares { get; set; }
    }
}
